src/Microsoft.Identity.Client/CacheV2/Schema/Credential.cs

```diff
@@ -229,18 +230,18 @@ public override string ToString()
         {
             var sb = new StringBuilder();
             sb.Append("Credential(\n");
-            sb.Append($"  HomeAccountId: {HomeAccountId}\n");
-            sb.Append($"  Environment: {Environment}\n");
-            sb.Append($"  Realm: {Realm}\n");
-            sb.Append($"  CredentialType: {CredentialType}\n");
-            sb.Append($"  ClientId: {ClientId}\n");
-            sb.Append($"  FamilyId: {FamilyId}\n");
-            sb.Append($"  Target: {Target}\n");
-            sb.Append($"  CachedAt: {CachedAt}\n");
-            sb.Append($"  ExpiresOn: {ExpiresOn}\n");
-            sb.Append($"  ExtendedExpiresOn: {ExtendedExpiresOn}\n");
-            sb.Append($"  Secret: {Secret}\n");
-            sb.Append($"  AdditionalFieldsJson: {AdditionalFieldsJson}\n");
+            sb.Append(string.Format(CultureInfo.InvariantCulture, "  HomeAccountId: {0}\n", HomeAccountId));
```

Comment:
Will all of our string formats require culture info going forward?

Reply:
Yes, sadly string interpolation is just syntactic sugar for String.Format, but without specifying Culture. The old FxCop, which we use for compliance, is actually running on an assembly, not on source code, so it can't even tell the difference between $(string) and String.Format. 

All in all, in product assemblies - don't use string interpolation. Should be ok in test assemblies, since on test assemblies we only run the new FxCop rules, which run on source code.

Reply:
Curious:  by default, is it invariant culture or is it current ui culture for interpolated strings?  it would be great if we could use string interpolation for logging and other messages and use string format only when we need real ui culture information.  i'm guessing we can't, but thought i'd ask.

Reply:
I actually checked string.cs and [stringbuilder](https://referencesource.microsoft.com/#mscorlib/system/text/stringbuilder.cs,947).cs and I see that by default they use CurentCulture

I really hate that we can't use string interpolation because of the old "FxCop", but I can't think of a way around it, bar ignoring rule CA1307 altogether.